### PR TITLE
CI: Bump Crystal version matrix

### PR DESCRIPTION
**Pull request by SamantazFox**

### CODE REVIEW

**Comment by ChunkyProgrammer** [comment]
Should the dockerfiles be bumped too (they point to 1.8.2)?

**Comment by SamantazFox** [comment]
@ChunkyProgrammer indeed, thanks :)

File: docker/Dockerfile.arm64 [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM alpine:3.18 AS builder
-RUN apk add --no-cache 'crystal=1.8.2-r0' shards sqlite-static yaml-static yaml-dev libxml2-static zlib-static openssl-libs-static openssl-dev musl-dev xz-static
+FROM alpine:3.19 AS builder
+RUN apk add --no-cache 'crystal=1.10.1-r0' shards sqlite-static yaml-static yaml-dev libxml2-static zlib-static openssl-libs-static openssl-dev musl-dev xz-static
```
Comment (syeopite) [inline docker/Dockerfile.arm64:2]: Is crystal 1.12 not supported on Alpine ARM yet?

Comment (SamantazFox) [reply docker/Dockerfile.arm64:2]: Not on 3.19, no. It's on Edge, but the container would break almost as soon as a new Crystal release is available.

**Review by syeopite — APPROVED** [review]

**Comment by SamantazFox** [comment]
As for testing, I've always been using the latest Crystal version on the test instance (except for 1.12.x) and it has been working great!

**Review by unixfox — APPROVED** [review]

**Pull request merged**